src/components/addSourceWizard/index.js

```diff
@@ -144,6 +144,7 @@ const AddSourceWizard = ({
       afterSuccess={afterSuccess}
       sourceTypes={state.sourceTypes}
       activeCategory={activeCategory}
+      sourceUuid={createdSource?.uid}
```

Comment:
Maybe a typo? `uid -> uuid`? just checking.

Reply:
So the object that holds the data indeed has the field as`uid,` I only realized it after i had already named all the variables passed around 😅 

Can have them all be uid if you'd prefer keeping consistency

Reply:
I see. We have a release coming we can fix it later.